pkgs/development/python-modules/mung/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, fetchgit
+# , fetchPypi
+, buildPythonPackage
+, pytest
+, lxml
+, scikitimage
+, scikitlearn
+, matplotlib
+, numpy
+, midiutil
+}:
+
+buildPythonPackage rec {
+  pname = "mung";
+  version = "unstable-2021-04-23";
+
+  src = fetchgit {
+    url = "https://github.com/OMR-Research/mung.git";
+    rev = "8fc816177e15889bd397eb20725441dbe21b8620";
+    sha256 = "1sann02126aas61f67kn4038raaj9arwq302nmi6vihsawy40sm5";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "OMR-Research";
    repo = "mung"
    rev = "8fc816177e15889bd397eb20725441dbe21b8620";
    sha256 = "1sann02126aas61f67kn4038raaj9arwq302nmi6vihsawy40sm5";
  };
```

Is there a tag for this revision?

---

No, and the last release on Pypi was ages ago.